com.vrcfury.vrcfury/SPS/sps_search.cginc

```diff
@@ -24,9 +24,15 @@ void sps_parse_light(float range, half4 color, out int type, int myChannel) {
 	// 		type = 0;
 	// 	}
 	// } else {
-		if (legacyRange == 1) type = 1;
-		if (legacyRange == 2) type = 2;
-		if (legacyRange == 5) type = 3;
+		if (myChannel == -1) { // legacy DPS channel 1
+			if (legacyRange == 3) type = 1;
+			if (legacyRange == 4) type = 2;
+			if (legacyRange == 6) type = 3;
+		} else { // Default Channel
+			if (legacyRange == 1) type = 1;
+			if (legacyRange == 2) type = 2;
+			if (legacyRange == 5) type = 3;
+		}
 	// }
 
 	if (channel != myChannel) {
```

Comment:
Just having a quick look, but wouldn't this condition always fail with DPS channel 1?
`myChannel`  would be `-1`, but `channel` is initialised to 0 on line 15, and never updated

Unless I missed something and this works regardless?